python/tvm/relay/op/_transform.py

```diff
@@ -182,6 +182,12 @@ def compute_unique(attrs, inputs, output_type):
 _reg.register_strategy("invert_permutation", strategy.invert_permutation_strategy)
 _reg.register_shape_func("invert_permutation", False, elemwise_shape_func)
 
+
+# einsum
+_reg.register_strategy("einsum", strategy.einsum_strategy)
+_reg.register_shape_func("einsum", False, elemwise_shape_func)
```

Comment:
`elemwise_shape_func` is incorrect I believe as the output shape can be pretty arbitrary. 

I don't think you need a shape func registered (I believe this is only used for dynamic inputs)

Reply:
NumpyEinsumShape might be a lead you want to explore

Reply:
I'll just remove the shape registration line for now, if we ever encounter a model with einsum shape dynamism I'll take a look at this again